server/utils/tabs.ts

```diff
@@ -0,0 +1,12 @@
+export interface TabsArgs {
```

Comment:
This is a TypeScript type for the object that we pass to the `govukTabs` macro - a subset of the "Nunjucks macro options" table [here](https://design-system.service.gov.uk/components/tabs/) - is that right?

I think it's an interesting idea to have strong types for these things - reduces the changes of an accidental typo. We haven't done it in any of other views so far so it would probably be good to create a tech debt for doing it consistently.

I wonder about the naming, though. I think we could be more explicit that these are related to GOV.UK Frontend macros. Perhaps `server/utils/govukFrontendTypes.ts`, and pop everything inside a [`namespace GovukFrontend`](https://www.typescriptlang.org/docs/handbook/namespaces.html)?

Reply:
Yeah, that's right. I've moved this into `server/utils/govukFrontendTypes.ts` for our benefit, so we can re-use it, and will add an issue to the tech debt list.

It looks like namespaces are no longer the "done" thing in modern Typescript ([at least according to the ESLint folks](https://github.com/typescript-eslint/typescript-eslint/blob/v4.6.1/packages/eslint-plugin/docs/rules/no-namespace.md)), so I've left it as a place where we can drop these things as exportable interfaces for now, but let me know if there's anything else you'd prefer.